tools/vf-component-library/src/site/_includes/layouts/tokens.njk

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/base.njk
+pageClass: posts
+templateEngineOverride: njk, md
+---
+
+{# Use this template for main sections of a site. #}
```

Comment:
Should be "Use this template for vf-tokens"

Reply:
resolved in last commit